config/stack/manifests/resources/bucket.ui-schema.yaml

```diff
@@ -1,4 +1,2 @@
-version: 0.4
+version: 0.5
```

Comment:
Leaving comment here for any other reviewers. The `version` of ui-schema is independently versioned from both the stack and the `apiVersion` of the resources. That being said, it is likely that a change in the `apiVersion` will affect the ui-schema in many situations. It would be nice to develop a check to keep these in sync in the future.